src/cli.ts

```diff
@@ -125,6 +125,8 @@ sade('lighthouse-parade <url>', true)
 
     const includePathGlob: string[] = toArray(opts['include-path-glob']);
 
+    const formFactor = opts['form-factor'];
```

Comment:
Could you also add documentation for this feature so that if someone does `lighthouse-parade --help` it lists the new flag and describes what it does? It would need to be added in these places:

1. Before line 80 in `cli.ts` (this sets the help text when `--help` is used)
2. In `README.md`

Thanks!

Reply:
You can use the description here from lighthouse-cli as a starting point: https://github.com/GoogleChrome/lighthouse#cli-options